pyartcd/pyartcd/pipelines/scheduled/schedule_build_conforma_verify.py

```
@@ -0,0 +1,100 @@
+                assembly='stream',
+                ec_policy=ec_policy,
+                fbc_ec_policy=fbc_ec_policy,
+                effective_time=effective_time,
+                include_corresponding_bundles=True,
+                include_corresponding_fbcs=True,
+                report_to_slack=True,
+                block_until_building=True,
+                block_until_complete=True,
+            )
+            runtime.logger.info('[%s] Conforma verify completed with result: %s', version, result)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)